src/test/pom-parser/pom-parser.test.js

```diff
@@ -73,4 +97,111 @@ describe('POMParser', () => {
         });
     });
   });
+
+  it('Child group id inheritance', () => {
+    const rawContent = readFile('child-group-id-inheritance.xml');
+
+    return pomParser.parsePOMFromString(rawContent)
+      .then((pomContent) => {
+          assert.equal(pomContent.groupId, 'a');
+      });
+  });
+
+  describe('TestRepoValues-parsePomFromString', () => {
+    it('repo-required', () => {
```

Comment:
What does it mean to indicate that the repo is required?  Are you indicating that the only thing being specified is the required values?